.github/workflows/scripts/homebrew.sh

```diff
@@ -50,13 +51,18 @@ function install_arm64() {
 }
 
 function install_intel() {
+    export CPU="x86_64"
```

Comment:
CPU? Where is that env used?

Reply:
Aha, in the Makefile

Reply:
righto, everything builds cleaner now, no need for the "just find a binary and name it 'this'" stuff anymore but meh.  ostensibly we could also make this a universal package by updating the run script, which we can think about.

Reply:
Ah, but then it shouldn't be in the install_* functions, as those only call brew, and brew only install precompiled packages.